modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -68,13 +68,25 @@ def allow_processing?
     end
 
     def va_gov_errored_claims
-      va_gov = ClaimsApi::AutoEstablishedClaim.select(:id, :transaction_id)
-                                              .where(created_at: @search_from..@search_to,
-                                                     status: 'errored', cid: '0oagdm49ygCSJTp8X297')
-                                              .group(
-                                                :id, :transaction_id
-                                              )
-      va_gov.group_by(&:transaction_id)
+      errored_claims = get_unique_errors
+      errored_claims&.group_by(&:transaction_id)
+    end
+
+    def get_unique_errors
+      starting = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+
+      one_day = ClaimsApi::AutoEstablishedClaim.where(created_at: 24.hours.ago..@search_to,
+                                                      status: 'errored', cid: '0oagdm49ygCSJTp8X297')
+
+      unique = one_day.uniq { |claim| claim[:transaction_id] } if one_day.present?
+      errored_claims = unique.select { |claim| claim[:created_at] >= @search_from } if unique.present?
```

Comment:
I _think_ the idea is to get the errored claims from the past 30 minutes and remove any that appear in the `24 hours ago` _up to_ `30 minutes ago` range (by transaction id), the logic being that we must have already reported on them.

As written, we will report all unique errored claims in the past hour, which could result in duplicate reported claims on successive runs.